README.md

```diff
@@ -30,9 +26,11 @@ This portfolio site must have the following HTML files:
 
 - `index.html` must include your name somewhere.
 
-- `about.html` should include some information about you, your interests, background or similar. Only post what you are comfortable sharing.
+- `facts.html` should include a series of fun facts or more detailed information about your topic.
 
-- `portfolio.html` must include information about several projects you have completed at Ada (or elsewhere) with links to the GitHub repo (if available), descriptions, images, etc.
+- `gallery.html` must include information about several projects you have completed at Ada (or elsewhere) with links to the GitHub repo (if available), descriptions, images, etc.
```

Comment:
To better reflect what we say the Gallery is at the top of the README, would the following work:
"must include at least 3 images with descriptions and links to where the images came from if applicable."